ultra/train.py

```diff
@@ -146,13 +152,28 @@ def train(task, num_episodes, policy_class, eval_info, timestep_sec, headless, s
     parser.add_argument(
         "--seed", help="environment seed", default=2, type=int,
     )
+    parser.add_argument(
+        "--log-dir", help="log directory location", default="logs", type=str,
```

Comment:
Maybe make the description start with an uppercase? Something like:
```suggestion
        "--log-dir", help="Log directory location", default="logs", type=str,
```
to match what Alex has in #487.

Reply:
That sounds good, i thought when we merge it, Alex's changes will overwrite mine, I guess